### ISSUE TITLE
Does not work React Native 0.62

### ISSUE THREAD

**Issue by RageOfJustice** [zoho/SalesIQ-Mobilisten-ReactNative#3]
# System info
System:
    OS: macOS 10.15.4
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 570.91 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 13.11.0 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.13.7 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 29
      Build Tools: 29.0.3
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6392135
    Xcode: 11.4.1/11E503a - /usr/bin/xcodebuild

# Project info
react-native: 0.62

enabled:
![image](https://user-images.githubusercontent.com/18666217/82208518-5a346400-9914-11ea-84da-0a0452867934.png)

pods installed according to 0.62 RN automatic-installation

# Steps to reproduce 

1. `npx react-native init somename`
2. `yarn add react-native-zohosalesiq-mobilisten`
3. `cd ios && pod install`
4.  Yes - for embed swift
5. `yarn ios --device` (--device because it does not build for [simulators](https://github.com/zoho/SalesIQ-Mobilisten-ReactNative/issues/1)) 

After these steps you can catch the error in xcode
```
dyld: Library not loaded: @rpath/Mobilisten.framework/Mobilisten
  Referenced from: /private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/BahamasEvac
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten: code signature invalid for '/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten'

	/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten: code signature invalid for '/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten'

	/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten: stat() failed with errno=1
	/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten: code signature invalid for '/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten'

	/private/var/containers/Bundle/Application/02C28F75-CC97-4426-92C1-4F77CFC80AF3/BahamasEvac.app/Frameworks/Mobilisten.framework/Mobilisten: stat() failed with errno=1
(lldb) 
```

**Comment by Rishabh-Raghunath** [comment]
This issue seems to be very similar to an bug in iOS 13.3.1 which causes a runtime crash in projects using external frameworks when using a free developer account.
Do you happen to be using a free developer account?
If yes, could you try updating your device to iOS 13.4.1 and check if the reported issue still happens?
Please try a clean and build from XCode as well to check if it solves the issue. 
Code signing issues are usually account related.

**Comment by RageOfJustice** [comment]
Ok Ill update to 13.4.1 and try again. 
But I do not understand how signing relates to this issue.
Could you provide documentation/article links from Apple where they says something like this?
Also its very convenient that I cannot test on the simulator (and very odd also) because my simulator device's version is up to date

**Comment by Rishabh-Raghunath** [comment]
Yes, you can find references to the issue which I have mentioned below.

- https://forums.developer.apple.com/thread/128987
- https://forums.developer.apple.com/thread/128987
- https://stackoverflow.com/questions/60096258/library-not-loaded-rpath-fblpromises-framework-fblpromises-ios-13-3-1
- https://stackoverflow.com/questions/60015309/how-to-run-an-ios-app-that-causes-runtime-error-for-frameworks-code-signature-i

This issue seems to have been resolved in an update to iOS 13.4.1 and hence wanted to confirm with you if this issue might be the underlying cause for the runtime crash.

**Comment by RageOfJustice** [comment]
Its working on the 13.4.1
